discord/user/user.py

```diff
@@ -10,36 +10,21 @@
 
 class User(BaseUser):
 
-    __slots__ = ("_stored",)
+    _stored: bool = False
 
-    _stored: bool
-
-    def __init__(self, cache, payload: UserPayload):
-        super().__init__(cache, payload)
-        self._stored = False
-
-    @classmethod
-    def _from_user(cls, user: BU) -> BU:
-        self = super()._from_user(user)
-        self._stored = False
-        return self
-
-    def __repr__(self) -> str:
-        return f"{self.tag} ({self.id})"
+    @property
+    def mutual_guilds(self):
+        return
 
-    def __str__(self) -> str:
-        return self.tag
+    async def create_dm(self):
+        pass
 
-    def __del__(self):
-        if self._stored:
-            del self._cache[self.id]
-        else:
-            raise KeyError
+    async def fetch_message(self):
+        pass
 
-    @property
-    def dm_channel(self):
-        return self._cache.get_user_dms(self)
+    async def edit(self, *, username: str = None, avatar: bytes = None):
```

Comment:
One cannot edit another user. You can only edit a member, or the ClientUser (the bot itself). This method should only be in those 2 classes.